src/DynamoCore/DynamoCore.csproj

```diff
@@ -431,6 +427,12 @@ limitations under the License.
     <Copy SourceFiles="@(LegacyBinariesToNodesFolder)" DestinationFolder="$(OutputPath)\nodes\" />
   </Target>
   <Target Name="AfterBuild" Condition=" '$(OS)' != 'Unix' ">
+    <ItemGroup>
+      <ASMLibG Include="$(SolutionDir)..\extern\LibG_*\*" />
+      <ExternProtoGeometry Include="$(SolutionDir)..\extern\ProtoGeometry\*" />
+    </ItemGroup>
+    <Copy SourceFiles="@(ASMLibG)" DestinationFolder="$(OutputPath)\%(RecursiveDir)" />
```

Comment:
As commented above, I don't think this folder is required anymore as we are getting all the necessary files from the nuget package install location.

Reply:
Answered on previous comment: https://github.com/DynamoDS/Dynamo/pull/11049#discussion_r476934753

Reply:
> I don't think this will work on dev machine setup.

Same comment here.

Reply:
Same answer here https://github.com/DynamoDS/Dynamo/pull/11049#discussion_r477428595